webrepl_cli.py

```diff
@@ -23,9 +23,12 @@
 
 
 def debugmsg(msg):
-    if DEBUG:
+    if OUTPUT_LEVEL > 1:
         print(msg)
 
+def infomsg(msg):
```

Comment:
No need to add this function.

Reply:
I'm afraid 
```python
if 1:
    print("op:%s, host:%s, port:%d, passwd:%s." % (op, host, port, passwd))
    print(src_file, "->", dst_file)
```
is a kind of bad code. But if you insist to leave it, OK then. Or maybe we could add a parameter to `debugmsg()` ?